_sass/fix-analytics/dialog.scss

```diff
@@ -0,0 +1,45 @@
+.dialog {
+  max-width: 560px;
+  width: 100%;
+  max-height: 100%;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border-radius: $border-radius;
+  box-shadow: $overlay-shadow, $plastic-material-shadow;
+  border: none;
+  background-color: white;
+  padding: 0;
+
+  @media(min-width: $width-md) {
+    max-height: 560px;
+  }
+}
+
+.dialog__body {
+  padding: 1rem 1.25rem;
+}
+
+.dialog__actions {
+  padding: 1rem 1.25rem;
+  position: sticky;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  justify-content: flex-end;
+  background-color: white;
+  border-top: 1px solid $border-grey;
+  gap: 10px;
+}
```

Comment:
Ville det være muligt at genbruge fra card, så dialog bare er en ekstra-class.

Altså at vi laver `class="dialog card"` i stedet for bare `class="dialog"`? Ligner at en del af koden går igen.

Reply:
Ja, det er rigtigt at det går igen det meste. Jeg tænke for/imod igennem og besluttede at jeg synes det var bedst med en class der kunne stå selv. Ellers skal man huske at teste både card og dialog hvis man ændre i card, og nu kan de divergerer.

Med brug af vores "styling"-variabler som $border-radius, $overlay-shadow osv, så er min tanke at looket på siden burde være ensartet, på trods af at der er forskellige classes. Til gengæld har vi mere duplikeret kode.

`padding: 1rem 1.25rem;` bliver brugt flere steder, så det kunne vi godt gemme som en variabel også.